modules/ossm-release-notes-3-0-3.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/ossm-release-notes/ossm-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-3-0-3_{context}"]
+= {SMProductName} version 3.0.3
+
+This release of {SMProductName} is included with the {SMProductName} Operator 3.0.3 and is supported on {ocp-product-title} 4.14 and later. This release addresses Common Vulnerabilities and Exposures (CVEs) and is supported on {ocp-product-title} 4.14 and later. For supported component versions for 3.0.2, see "Service Mesh version support tables".
+
+[id="ossm-enhancements-3-0-3_{context}"]
+== Enhancements
+
+* https://issues.redhat.com/browse/OSSM-8608[OSSM-8608] This enhancement updates {istio} to version 1.24.6.
```

Comment:
Seems wrong issue link, suppose that should be https://issues.redhat.com/browse/OSSM-9758